diff --git a/meson.build b/meson.build
index 2d9200529ca..d4a08865388 100644
--- a/meson.build
+++ b/meson.build
@@ -175,7 +175,7 @@ if gallium_drivers.contains('auto')
             host_machine.cpu_family()))
     endif
   elif ['windows'].contains(host_machine.system())
-    gallium_drivers = ['llvmpipe', 'softpipe', 'zink', 'd3d12']
+    gallium_drivers = ['llvmpipe']
   elif ['darwin', 'cygwin', 'haiku'].contains(host_machine.system())
     gallium_drivers = ['llvmpipe', 'softpipe']
   else